packages/mermaid-zenuml/src/zenumlRenderer.ts

```diff
@@ -0,0 +1,89 @@
+// @ts-nocheck TODO: fix file
+import { getConfig, log } from './mermaidUtils';
+import ZenUml from 'vue-sequence';
+import { regexp } from './detector';
+import 'vue-sequence/dist/vue-sequence.css';
+
+// Load Zen UML CSS
+function loadCss(root: Document, url: string) {
+  // Avoid loading the same CSS multiple times
+  if (root.querySelector(`link[href="${url}"]`)) {
+    return;
+  }
+
+  const link = root.createElement('link');
+  link.type = 'text/css';
+  link.rel = 'stylesheet';
+  link.href = url;
+  root.getElementsByTagName('head')[0].appendChild(link);
+}
+
+// Create a Zen UML container outside of the svg first for rendering, otherwise the Zen UML diagram cannot be rendered properly
+function createTemporaryZenumlContainer() {
+  const container = document.createElement('div');
+  container.innerHTML = '<div id="app"></div>';
+  container.style.display = 'none';
+  const app = container.querySelector('#app') as HTMLElement;
```

Comment:
```suggestion
  container.innerHTML = '<div id="zenUMLApp"></div>';
  container.style.display = 'none';
  const app = container.querySelector('#zenUMLApp') as HTMLElement;
```

`app` seems too generic. How about `diagram` or `zenuml` ?